apps/test-bundles/webpackUtils.js

```diff
@@ -73,6 +74,47 @@ function createWebpackConfig(entries) {
 // Files which should not be considered top-level entries.
 const TopLevelEntryFileExclusions = ['index.js', 'version.js', 'index.bundle.js'];
 
+function createEntries(packageName, isNorthstar) {
+  const replacePath = isNorthstar ? ['commonjs', 'es'] : ['lib-commonjs', 'lib'];
+  const distPath = path.dirname(require.resolve(packageName).replace(replacePath[0], replacePath[1]));
+  const packagePath = path.resolve(distPath, isNorthstar ? 'components' : '');
+  fs.readdirSync(packagePath).forEach(itemName => {
+    const isFolder = fs.statSync(path.join(packagePath, itemName)).isDirectory();
+    const isAllowedFile = itemName.match(/.js$/) && !TopLevelEntryFileExclusions.includes(itemName);
+
+    if ((!isAllowedFile && !(isNorthstar && isFolder)) || !itemName) {
+      return;
+    }
+
+    const item = isFolder ? itemName : itemName.replace(/.js$/, '');
+    const importStatement = `import { ${item} } from '${packageName}'; console.log(${item})`;
+    try {
+      const folderName = getFolderName(packageName);
+      if (!fs.existsSync(`fixtures/${folderName}`)) {
```

Comment:
We have `fs-extra` package already in dependencies, https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/outputFile-sync.md might be usefull